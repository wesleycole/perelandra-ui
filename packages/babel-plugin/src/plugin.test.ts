import pluginTester from 'babel-plugin-tester'
import plugin from '.'

pluginTester({
  plugin: plugin,
  pluginName: 'perelandra-ui',
  snapshot: true,
  tests: [{ code: '"hello";' }],
})
