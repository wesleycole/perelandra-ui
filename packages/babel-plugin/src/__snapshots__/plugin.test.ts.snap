// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`perelandra-ui 1. perelandra-ui: 1. perelandra-ui 1`] = `

"hello";

      ↓ ↓ ↓ ↓ ↓ ↓

'hello'


`;
